packages/client/src/library/Svg/index.js

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from "react";
+
+const Svg = ({ name, ...props }) => {
+  const [svgComponent, setSvgComponent] = useState(null);
+  useEffect(() => {
+    (async () => {
+      try {
+        const svgComponent = await import(`./${name}`);
+        setSvgComponent(svgComponent.default(props));
+      } catch (e) {
+        console.log(e);
+      }
+    })();
+    //eslint-disable-next-line
+  }, [name]);
+  if (!svgComponent) return null;
```

Comment:
`svgComponent` should already be null if component fails to load so this line shouldn't be necessary